python/services/evaluationservice/dmod/evaluationservice/evaluation_service/apps.py

```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class EvaluationServiceConfig(AppConfig):
+    name = 'evaluation_service'
```

Comment:
Not sure where exactly to insert this comment, so just picking here.  We probably need to move (or at least copy) all the Django related stuff out of this service, although I'm for having a larger group discussion first.  

The general design of the `main` DMOD services stack has been to treat all the services - except for the _requestservice_ - as though they are only reachable from other services inside the stack.  The containers are only attached to private Docker networks, except for _requestservice_, which is assumed to be the point of entry for anything "external."  

Further, we decided to treat the GUI as something external, despite also providing one, to support allowing externally/separately developed GUIs to attach to a DMOD deployment.  The result is that the `main` stack behaves like a websocket-based API endpoint, with any external communication following a specified protocol defined via the different `dmod.communication.Message` subtypes.

Now, @hellkite500, @stcui007, and I had these design discussions a pretty long time ago, so we probably _need_ to considering if parts of the approach should be adjusted (I'll ping @mattw-nws for that too).  But until we decide if and how to change things, the evaluation service just needs to run a customized implementation of `dmod.communication.WebSocketInterface`.  This should listen for and handle websocket connections, through which various serialized evaluation-related request and response message objects are sent.

We will want all the Django stuff _somewhere_, just not here (at least for the moment).

Reply:
Some of the structure was based off of some of the structure of the subset service, I believe (though that uses flask instead of Django). One of the challenges with `dmod.communication.WebSocketInterface` is that the most prevalent type of communication with the evaluation service is through basic REST calls, not sockets. It also looked like `dmod.communication.Message` (if that's the one I remember right) makes a lot of assumptions about class deserialization whereas all of the messages passed through the eval service have very simple and/or dynamic json payloads or simply url based parameters. It's pretty murky.

The overall plan was to separate the evaluation service from the evaluations library; anything that's not affiliated with communication (url routing, socket consumer, templating, views, static page resources, etc) should be in the evaluations library. Even the lone socket consumer wouldn't really fit with processing logic - it just forwards a redis channel through a websocket. The worker _does_ live in the service, so maybe that can move over to evaluations?

Reply:
Scratch `worker.py` moving to another module - it needs to live with the `runner.py`, who'll be calling it out of process. Maybe something clever can be done about that.